### PR TITLE
Added Italian translation.

### PR DESCRIPTION
**Pull request by pfmaggi**
Simple Italian translation.

### CODE REVIEW

**Comment by pfmaggi** [comment]
Funny, the translations editor doesn't play well with the plurals (it doesn't show them at all).

I've added the missing strings.

File: app/src/main/res/values-it/strings.xml [hunk 1]
```diff
@@ -14,16 +14,16 @@
...
-    <string name="watering_needs_prefix" translation_description="Prefix for the text describing how often a plant should be watered. Used in combination with watering_needs_suffix.">Watering needs:</string>
+    <string name="watering_needs_prefix" translation_description="Prefix for the text describing how often a plant should be watered. Used in combination with watering_needs_suffix.">Esigenze di irrigazione:</string>
     <plurals name="watering_needs_suffix">
-        <item quantity="one" translation_description="Suffix for the text describing how often a plant should be watered, singular. Used in combination with watering_needs_prefix: 'Watering needs: every day'.">every day</item>
-        <item quantity="other" translation_description="Suffix for the text describing how often a plant should be watered, plural. Used in combination with watering_needs_prefix: 'Watering needs: every 8 days'.">every %d days</item>
+        <item quantity="one" translation_description="Suffix for the text describing how often a plant should be watered, singular. Used in combination with watering_needs_prefix: 'Watering needs: every day'.">ogni giorno</item>
```
Comment (XinyueZ) [inline app/src/main/res/values-it/strings.xml]: I dont think `translation_description` is really needed, would you mind to remove it?

Comment (pfmaggi) [reply app/src/main/res/values-it/strings.xml]: Thanks, I've removed `translation_description` from the italian strings.xml file.

**Comment by pfmaggi** [comment]
Added some missing strings, added copyright notice and removed `translation_description`.

File: app/src/main/res/values-it/strings.xml (hunk 1 above)
Comment (XinyueZ) [reply app/src/main/res/values-it/strings.xml]: thx

File: app/src/main/res/values-it/strings.xml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+<!--
+  ~ Copyright 2018 Google LLC
```
Comment (XinyueZ) [inline app/src/main/res/values-it/strings.xml]: 2019 
I had same issue before :)

Comment (pfmaggi) [reply app/src/main/res/values-it/strings.xml]: Sorry... time travel... :-D

**Comment by nic0lette** [comment]
Thank you again! =D

**Pull request merged**